package.json

```diff
@@ -44,31 +47,36 @@
     "url": "https://github.com/gnosis/gnosis-safe/issues"
   },
   "devDependencies": {
+    "@elvis-krop/hardhat-deploy": "0.11.29-ethers5.5.0",
```

Comment:
What is the difference of this to the normal hardhat-deploy?

Reply:
Hello @rmeissner :wave: 
We are waiting for an action under [this PR in hardhat-deploy](https://github.com/wighawag/hardhat-deploy/pull/447). You can read more details in the description.

> During the development, we realised that [v0.11.25](https://github.com/wighawag/hardhat-deploy/tree/v0.11.25) and previous versions doesn't work properly for zkEVM deployments and testing. Moreover, v0.11.26 and further releases include incompatible versions of @ethersproject/* with safe-contracts repo ethers.js and some tools for testing.

We expected to have an intermediate release of hardhat-deploy for this, however owner doesn't reply us

Reply:
I doubt that they will be downgrading their ethers dependency. You can force a specific resolution of a dependency with yarn afaik (see https://classic.yarnpkg.com/en/docs/selective-version-resolutions/)

Reply:
Thank you @rmeissner!
:heavy_check_mark: done at [b492bce](https://github.com/safe-global/safe-contracts/pull/588/commits/b492bce6cb554e51d96f531640c2e72d65c77162)